components/Footer/Footer.js

```diff
@@ -3,12 +3,24 @@ import styled from 'styled-components'
 import FooterContentArea from './FooterContentArea'
 import Map from './Map'
 import Link from 'next/link'
+import GoogleLogoPNG from '../../assets/png/footer/google-logo.png'
 import { useMediaQuery } from 'react-responsive'
 
 export default function Footer() {
   const isDesktop = useMediaQuery({ query: '(max-width: 841px)' })
   return (
     <Wrapper>
+      <GoogleRezession>
+        <p>
+          Waren Sie mit unser Leistung zufrieden oder haben Sie Vorschläge, wie
+          wir uns verbessern können? Dann klicken Sie auf das <a style={{"color" : "white"}} href="https://www.google.com/search?q=nordlichter+halsenbek#lrd=0x47b18128d7d26c01:0x975243cc198cbb69,1,,," target="_blank">Google Logo</a> und
```

Comment:
To increase security when linking to another host, I would use the rel attribute.

```suggestion
          wir uns verbessern können? Dann klicken Sie auf das <a style={{"color" : "white"}} href="https://www.google.com/search?q=nordlichter+halsenbek#lrd=0x47b18128d7d26c01:0x975243cc198cbb69,1,,," target="_blank" rel="noopener noreferrer">Google Logo</a> und
```

Reply:
Das wusste ich nicht. Danke dir.